doc/source/frequently-asked-questions.rst

```diff
@@ -1,8 +1,8 @@
 :tocdepth: -1
 
-.. index:: frequently_asked_questions
+.. index:: frequently-asked-questions
```

Comment:
I realized that we may want to adopt 

`https://my-page/use-minus-sign-for-pages` instead of `https://my-page/user_undersocre_sign` 

ex) `https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site`

Reply:
yes, I agree.  I was moving in this direction with the bg-mpl-stylesheets and so on.  Getting rid of underscores in URLs and filenames, and CLI args (but replacing them obviously in python code) as a standard.

Reply:
> I realized that we may want to adopt
> 
> `https://my-page/use-minus-sign-for-pages` instead of `https://my-page/user_undersocre_sign`
> 
> ex) `https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site`

@sbillinge yup noted, I will just note this in the gitlab doc